futures-util/src/stream/mod.rs

```diff
@@ -1072,8 +1072,8 @@ pub trait StreamExt: Stream {
     /// resolve to [`None`] if used on an empty [`Stream`]. Instead, the
     /// returned future type will return [`true`] from
     /// [`FusedFuture::is_terminated`][] when the [`Stream`] is empty, allowing
```

Comment:
Looks like you're missing a link here

Reply:
It's below, the full path is very long so I found it best to keep it out of the prose.